### PR TITLE
Persist Wishes to the localstorage #4

### PR DESCRIPTION
**Pull request by Eth3R-One**
Added feature to add new wish to local storage and get saved wishes from local storage.

### CODE REVIEW

File: src/AddToBucket.jsx [hunk 1]
```diff
@@ -3,6 +3,17 @@ import { createSignal } from "solid-js";
...
 
+  //  Save new wish to localstorage
+  function saveToLocalStorage(prev, newWish){
+    const allWishes = [
+      ...prev, {
+        text: newWish,
+        complete:false,
+      }
+    ].reverse()
+    localStorage.setItem("solid-bucket-list", JSON.stringify(allWishes))
```
Comment (atapas) [inline src/AddToBucket.jsx:14]: @Eth3R-One Let's create a utility file for the lcalstorage related operations... u can create a util folder under src and create the utility under that.

**Comment by Eth3R-One** [comment]
Added new utility file for local storage related operations. added two functions to get and save wishes at this moment
src/
  |__util/
            |__localStorageUtility.js

**Comment by sarowarhosen03** [comment]
hello sir
i have done this can i make a pR

**Comment by Eth3R-One** [comment]
@sarowarhosen03 Yes I've made the changes and waiting for review and approval.

**Comment by sarowarhosen03** [comment]
> @sarowarhosen03 Yes I've made the changes and waiting for review and approval.

Oh okay thanks for letting me know

**Comment by atapas** [comment]
@Eth3R-One Could you please resolve the merge conflicts?

**Comment by Eth3R-One** [comment]
@atapas Dada I resolved the conflicts. Let me know if there is any issue

File: src/util/localStorageUtil.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ *  return saved wishes data from local-storage.
+ *  if no data is saved to localstorage then return dummy data
+ */
+export function getWishes() {
+  const wishes = JSON.parse(localStorage.getItem("solid-bucket-list"));
+
+  return wishes ? wishes : dummy_Data;
+  //   if (!wishes) return dummy_Data;
```
Comment (atapas) [inline src/util/localStorageUtil.js:14]: remove the commented lines

**Review by atapas — CHANGES_REQUESTED** [review]

**Review by atapas — APPROVED** [review]

**Comment by atapas** [comment]
@all-contributors please add @Eth3R-One  for code

**Comment by atapas** [comment]
@all-contributors please add @atapas for Infrastructure

**Pull request merged**